configs/swarm/genesis.json

```diff
@@ -127,43 +127,43 @@
         }
       },
       {
-        "NewParameter": "?MaxTransactionsInBlock=512"
+        "SetParameter": "?MaxTransactionsInBlock=512"
       },
       {
-        "NewParameter": "?BlockTime=2000"
+        "SetParameter": "?BlockTime=2000"
       },
       {
-        "NewParameter": "?CommitTimeLimit=4000"
+        "SetParameter": "?CommitTimeLimit=4000"
       },
       {
-        "NewParameter": "?TransactionLimits=4096,4194304_TL"
+        "SetParameter": "?TransactionLimits=4096,4194304_TL"
       },
       {
-        "NewParameter": "?WSVAssetMetadataLimits=1048576,4096_ML"
+        "SetParameter": "?WSVAssetMetadataLimits=1048576,4096_ML"
       },
       {
-        "NewParameter": "?WSVAssetDefinitionMetadataLimits=1048576,4096_ML"
+        "SetParameter": "?WSVAssetDefinitionMetadataLimits=1048576,4096_ML"
       },
       {
-        "NewParameter": "?WSVAccountMetadataLimits=1048576,4096_ML"
+        "SetParameter": "?WSVAccountMetadataLimits=1048576,4096_ML"
       },
       {
-        "NewParameter": "?WSVDomainMetadataLimits=1048576,4096_ML"
+        "SetParameter": "?WSVDomainMetadataLimits=1048576,4096_ML"
       },
       {
-        "NewParameter": "?WSVIdentLengthLimits=1,128_LL"
+        "SetParameter": "?WSVIdentLengthLimits=1,128_LL"
       },
```

Comment:
can we drop the `WSV` prefix? what is it's use?

Reply:
@0x009922

Reply:
Well, if we are speaking in the context of #4028, then... `SetParameter` should not accept some magically constructed string. It should accept a hard-coded-into-data-model enum of possible on-chain parameters.

As for prefix, I don't think it is needed. In the issue there are comments about naming.

Reply:
created #4325